dt-groups/groups-post-type.php

```diff
@@ -393,60 +424,76 @@ public function get_group_field_defaults( $post_id = null, $include_current_post
             'section'     => 'misc',
             'hidden'      => true
         ];
-        $fields["member_count"] = [
-            'name' => __( 'Member Count', 'disciple_tools' ),
-            'type' => 'text',
-            'default' => ''
-        ];
         $fields["parent_groups"] = [
-            "name" => __( "Parents", "disciple_tools" ),
+            "name" => __( "Parent Group", "disciple_tools" ),
+            'description' => 'A group that founded this group.',
             "type" => "connection",
             "p2p_direction" => "from",
             "p2p_key" => "groups_to_groups"
         ];
-        $fields["child_groups"] = [
-            "name" => __( "Child", "disciple_tools" ),
-            "type" => "connection",
-            "p2p_direction" => "to",
-            "p2p_key" => "groups_to_groups"
-        ];
         $fields["peer_groups"] = [
-            "name" => __( "Peer Groups", "disciple_tools" ),
+            "name" => __( "Peer Group", "disciple_tools" ),
+            'description' => "A related group that isn't a parent/child in relationship. It might indicate groups that collaborate, are about to merge, recently split, etc.",
             "type" => "connection",
             "p2p_direction" => "any",
             "p2p_key" => "groups_to_peers"
         ];
+        $fields["child_groups"] = [
+            "name" => __( "Child Group", "disciple_tools" ),
+            'description' => 'A group that has been birthed out of this group.',
+            "type" => "connection",
+            "p2p_direction" => "to",
+            "p2p_key" => "groups_to_groups"
+        ];
         $fields["members"] = [
             "name" => __( "Members", "disciple_tools" ),
+            'description' => 'Who is a member of this group?',
             "type" => "connection",
             "p2p_direction" => "to",
             "p2p_key" => "contacts_to_groups"
         ];
+        $fields["member_count"] = [
+            'name' => __( 'Member Count', 'disciple_tools' ),
+            'description' => 'The number of members in this group. The member count will be automatically adjust when new member names are added or removed in the member list. The number can be adjusted manually to included people who may not be in the system but are members of the group.',
+            'type' => 'text',
+            'default' => ''
+        ];
+        $fields["member_list"] = [
```

Comment:
The members list is already declared with the "members" field. 
This one is duplicate.